docs/fast_data/configuration/config_maps/projection_changes_schema.md

```diff
@@ -6,7 +6,14 @@ sidebar_label: Projection Changes Schema
 
 ## Overview
 
-The Projection Changes Schema is a JSON file (`projectionChangesSchema.json`) which helps the strategy to find the right path from the *Initial projection*\* to the *Base projection*\*\* and then to the identifier used to match the single view document. This file is typically used by the [Real Time Updater](/fast_data/realtime_updater.md) or the [Single View Trigger Generator](/fast_data/single_view_trigger_generator.md) to execute the strategies.
+The Projection Changes Schema is a JSON file (`projectionChangesSchema.json`) which helps the [strategy](/fast_data/the_basics.md#strategies) to find the right path from the *Initial projection*
+to the *Base projection* and then to the identifier used to match the single view document. This file is typically used
+by the [Real Time Updater](/fast_data/realtime_updater.md) or the [Single View Trigger Generator](/fast_data/single_view_trigger_generator.md) to execute the strategies.
+
+:::info
```

Comment:
let's add these concepts also in the Glossary page

Reply:
Done